var/spack/repos/builtin/packages/scr/package.py

```diff
@@ -33,7 +33,12 @@ class Scr(CMakePackage):
     # of a few component libraries
     depends_on('axl@master', when="@develop")
     depends_on('kvtree@master', when="@develop")
+    depends_on('er@master', when="@develop")
     depends_on('redset@master', when="@develop")
+    depends_on('rankstr@master', when="@develop")
+    depends_on('shuffile@master', when="@develop")
+    depends_on('filo@master', when="@develop")
```

Comment:
@robertkb , it's worth testing to be sure, but I think we can drop filo for @develop.
```suggestion
```

Reply:
And if that works, let's also drop the filo line for @3: for the same reason.

Reply:
There's no `version()` specified for scr for anything past 2.0.0. I suppose the intention is that a version 3.0.0 would be specified some time in the future. The colon in `@3:` means the dependency applies to future versions, but the `when="@3:"` lines currently don't do anything, since they are overridden by the `when="@develop"` spec. Should I remove them all, replace my `@develop` spec with corrected `@3:` specs, or leave them there?